safe/impact_functions/inundation/flood_polygon_population/test/test_flood_polygon_population.py

```diff
@@ -48,6 +48,8 @@ def test_run(self):
 
         function.hazard = hazard_layer
         function.exposure = exposure_layer
+        function.parameters['affected_field'] = 'FLOODPRONE'
```

Comment:
Dont you think we should be using keywords in the layer rather than parameters to define these?